pydantic/main.py

```diff
@@ -673,6 +681,13 @@ def _get_value(
                 and (not value_include or value_include.is_included(i))
             )
 
+        elif (
+            isinstance(v, Enum)
+            and getattr(cls.Config, 'use_enum_values', False)
+            and any(is_literal_type(hint) for hint in get_type_hints(cls).values())
```

Comment:
I'm not quite sure what this line is doing?

Surely if `isinstance(v, Enum) and getattr(cls.Config, 'use_enum_values', False)` is enough?

Reply:
It seems so, yes